### PR TITLE
better checking of distro name

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: package-configuration/linux-cli/get-cloudify.py [hunk 1]
```diff
@@ -263,20 +263,36 @@ def install_pip(self):
             sys.exit('Could not install pip')
 
     def install_pythondev(self):
+        """Installs python-dev and gcc
+
+        This will try to match a command for your distribution.
+        The reason for looping through the names of the distros instead of just
+        checking for equality is that the name of the distro `DISTRO` might not
+        always be a pure distro name. For instance, in CentOS 7.1,
+        platform.linux_distribution() return 'CentOS Linux'. Checking for the
```
Comment (m1keil) [inline package-configuration/linux-cli/get-cloudify.py]: not entierly correct, you just need to call platform.linux_distribution(full_distribution_name=False) to get the short name. https://docs.python.org/2/library/platform.html#platform.linux_distribution

Comment (m1keil) [inline package-configuration/linux-cli/get-cloudify.py]: also:
_supported_dists = (
    'SuSE', 'debian', 'fedora', 'redhat', 'centos',
    'mandrake', 'mandriva', 'rocks', 'slackware', 'yellowdog', 'gentoo',
    'UnitedLinux', 'turbolinux')

File: package-configuration/linux-cli/get-cloudify.py [hunk 2]
```diff
@@ -263,20 +263,36 @@ def install_pip(self):
...
+        """
         prn('Installing python-dev...')
-        if DISTRO in ('ubuntu', 'debian'):
-            cmd = 'apt-get install -y gcc python-dev'
-        elif DISTRO in ('centos', 'redhat'):
-            cmd = 'yum -y install gcc python-devel'
-        elif os.path.isfile('/etc/arch-release'):
+        cmd = ''
+        for distro in ('ubuntu', 'debian'):
+            if distro in DISTRO:
```
Comment (m1keil) [inline package-configuration/linux-cli/get-cloudify.py]: so, i suggest reverting it back, this for/if thing is confusing

Comment (nir0s) [inline package-configuration/linux-cli/get-cloudify.py]: wonderful idea. I didn't think about it :)

File: package-configuration/linux-cli/get-cloudify.py (hunk 1 above)
Comment (nir0s) [inline package-configuration/linux-cli/get-cloudify.py]: And?

**Comment by m1keil** [comment]
LGTM =)

**Pull request merged**